tests/nativeapp/test_manager.py

```diff
@@ -246,7 +247,10 @@ def test_drop_object_no_special_comment(mock_execute, temp_dir, mock_cursor):
                     ],
                     [],
                 ),
-                mock.call("show application packages like 'sample_package_name'"),
+                mock.call(
+                    "show application packages like 'SAMPLE_PACKAGE_NAME'",
+                    cursor_class=DictCursor,
+                ),
```

Comment:
The assert call below actually never got reached, which is why this had to change (even though the underlying code did not).